### DISCUSSION TITLE
i don't know why

### DISCUSSION THREAD

**Discussion by A1foostbit** [KadeArchive/Kade-Engine#540] [Q&A] [answered]
so I was building the game and it said this source/Character.hx:445: characters 5-16 : Unknown identifier : playAnimate

**Comment by A1foostbit** [comment, marked as answer]
nev

**Reply by ghost** [reply]
Choose your answer as the answer to make the discussion how as answered.

**Reply by puyoxyz** [reply]
anyone looking for the solution: its playAnim not playAnimate
